go.mod

```diff
@@ -28,9 +28,9 @@ require (
 	k8s.io/api v0.18.8
 	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	knative.dev/eventing v0.17.1-0.20200929080328-74b9bed2ebf2
-	knative.dev/pkg v0.0.0-20200929052028-236c084e37a3
-	knative.dev/serving v0.17.1-0.20200929093229-7960ee24cc7b
+	knative.dev/eventing v0.18.0
+	knative.dev/pkg v0.0.0-20200922164940-4bf40ad82aab
```

Comment:
this same thing happened before? this is because there is no 0.18.0 tag?

Reply:
It seems there is no tag yet:
```
go: knative.dev/pkg@v0.18.0: reading knative.dev/pkg/go.mod at revision v0.18.0: unknown revision v0.18.0
go: knative.dev/serving@v0.18.0: reading knative.dev/serving/go.mod at revision v0.18.0: unknown revision v0.18.0
```